### PR TITLE
KCARE-1152 Secure Boot detection

### PR DESCRIPTION
**Pull request by histrio**
- Also updated copyright due to KCARE-1200

Signed-off-by: Rinat Sabitov <rinat.sabitov@gmail.com>

### CODE REVIEW

**Comment by histrio** [comment]
@aborilov Could you please make a review?

**Review by aborilov — APPROVED** [review]
👍

File: py/kc-compat.py [hunk 1]
```diff
@@ -39,11 +39,7 @@ def _get_last_byte_from(filename):
     :return: last byte in a file as unsigned int or None if file was empty
     """
     with open(filename, 'rb') as f:
-        try:
-            f.seek(-1, os.SEEK_END)
-        except IOError:
-            return None
-        last, = struct.unpack("B", f.read(1))
+        last, = struct.unpack("B", f.read()[-1])
```
Comment (aborilov) [inline py/kc-compat.py:42]: do we sure that this file is not too big?

Comment (histrio) [reply py/kc-compat.py:42]: Pretty sure. It's always 5 bytes.
I've changed this part due to impossible doing `seek` in efivar files: it raises `OSError: Illegal seek` on any attempts to do any seek.
Thanks!

**Pull request merged**